### PR TITLE
Create api module and move GetDBEndpoints to it

### PR DESCRIPTION
**Pull request by karelyatin**
GetDBEndpoints will also be used by other operators like neutron, octavia etc.

### CODE REVIEW

File: api/v1alpha1/client.go [hunk 1]
```diff
@@ -14,14 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package client
+package v1alpha1
```
Comment (slawqo) [inline api/v1alpha1/client.go:17]: it's not for this change maybe but AFAIK all other operators starts with v1beta1 so maybe You should change it in ovn-operator too

**Review by slawqo — APPROVED** [review]

File: api/v1alpha1/client.go (hunk 1 above)
Comment (karelyatin) [reply api/v1alpha1/client.go:17]: Thanks slawek, if that's the case can do that in follow up.

**Review by abays — APPROVED** [review]
/lgtm

**Comment by karelyatin** [comment]
Thanks Andrew and Slawek, Merging it and will update neutron-operator PR.

**Pull request merged**